example/App.js

```diff
@@ -94,6 +94,15 @@ export const App = () => {
     });
   };
 
+  const sendLocalNotificationWithSound = () => {
+    PushNotificationIOS.presentLocalNotification({
```

Comment:
`presentLocalNotification` is now deprecated, could you use `addRequestNotification` instead? Should work the same way (use `sound` instead of `soundName` along with other prop names mentioned in README.